src/profiler.cpp

```diff
@@ -1528,6 +1528,7 @@ Error Profiler::runInternal(Arguments& args, std::ostream& out) {
         }
         case ACTION_DUMP: {
             Error error = dump(out, args);
+            logUsedMemory(args);
```

Comment:
Matter of taste but I'd do the `if` here and not in the function

Reply:
agreed, fixed